fe/fe-core/src/main/java/com/starrocks/load/routineload/KafkaRoutineLoadJob.java

```diff
@@ -831,6 +831,13 @@ public void modifyJob(RoutineLoadDesc routineLoadDesc, Map<String, String> jobPr
                     throw new DdlException("The specified partition is not in the consumed partitions ", e);
                 }
             }
+
+            // validate broker list if provided
+            try {
+                CreateRoutineLoadStmt.validateKafkaBrokerList(dataSourceProperties.getKafkaBrokerList());
+            } catch (AnalysisException e) {
+                throw new DdlException(e.getMessage(), e);
+            }
```

Comment:
better to validate in `RoutineLoadDataSourceProperties.checkKafkaProperties`

Reply:
Fixed